mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -178,6 +181,7 @@ SONIC:
 		Cost: 1300
 	Tooltip:
 		Name: Disruptor
+		Description: Advanced anti-armour vehicle.\n  Strong vs Infantry, Tanks\n  Weak vs Aircraft
```

Comment:
[23:57] <penev> isn't the disruptor more like an anti-defence tank?
[23:57] <penev> iirc it's great against buildings, has a long range, and dies like a fly to both infantry and vehicles